docs/reference/extensions.rst

```diff
@@ -6,20 +6,82 @@ Rockcraft extensions are crafted to expand and modify the user-provided
 rockcraft project file, aiming to minimise the boilerplate code when
 initiating a new rock.
 
+.. _flask-framework-reference:
```

Comment:
it might be better to make extensions a dir with an index and put flask into its own rst. This is cleaner when linking to documentation from tooling and concise in the sense that you can start reading from the top instead of visually searching for the anchor

Reply:
makes sense. done